httpserver/types.go

```diff
@@ -22,9 +22,11 @@ import (
 
 const (
 	VerificationResultVersion = "0.1.0"
+	ResultVersion0_2_0        = "0.2.0"
```

Comment:
do we have any documentation/Samples for each version of verification result?

Reply:
Yes, we have one for 0.1.0 and 1.0.0, I'll update 0.2.0 and 1.1.0 after this pr. https://ratify.dev/docs/next/reference/verification-result-version

Reply:
Discussed offline, will create an issue to track response version.